src/dart/install.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+set -e
+source _common.sh
+
+# TODO: Do we need to "check_packages" here?
+
+# https://dart.dev/get-dart
+# "Perform the following one-time setup:"
+sudo apt-get update
+sudo apt-get install apt-transport-https
+wget -qO- https://dl-ssl.google.com/linux/linux_signing_key.pub | sudo gpg --dearmor -o /usr/share/keyrings/dart.gpg
```

Comment:
Should check `wget` installed before this line.

Reply:
How about rewrite it to the same process as the official Feature?
https://github.com/devcontainers/features/blob/852f1f0567280b465913b487d5f8c773122c36ab/src/node/install.sh#L99-L103

Reply:
I tried to stay as close to the official https://dart.dev/get-dart way as I reasonably could (hence my earlier mistake with sudo 🤦‍♂️).

![image](https://github.com/devcontainers-community/features/assets/61068799/25239b59-40fc-4a36-8349-1036ed32f3f0)

I note specifically this part:
```sh
# Yarn example
echo "deb [arch=$(dpkg --print-architecture) ..."
# The official https://dart.dev/get-dart
echo 'deb ... arch=amd64]'
```

Is this significant? Are there other significant differences between https://dart.dev/get-dart and the way it's done in the Node.js yarn thing in the devcontainers/features repo?

Reply:
> Is this significant? Are there other significant differences between [dart.dev/get-dart](https://dart.dev/get-dart) and the way it's done in the Node.js yarn thing in the devcontainers/features repo?

This difference is very important and must be emphasized in the documentation because if Dart only supports amd64, the arm64 platform will not be able to use this Feature.